### PR TITLE
fix(pointcloud-pattern): Add more custom for switchPattern

### PR DESCRIPTION
**Pull request by Kerwin-L**

### CODE REVIEW

File: packages/lb-components/src/views/MainView/toolHeader/SwitchPattern/index.tsx [hunk 1]
```diff
@@ -64,12 +76,7 @@ const SwitchPattern = ({ toolName }: IProps) => {
       >
         {t('DetectionMode')}
       </Button>
-      <Button
-        type={globalPattern === EPointCloudPattern.Segmentation ? 'primary' : undefined}
-        onClick={updateSegmentation}
-      >
-        {t('SegmentationMode')}
-      </Button>
+      {forbidSegmentation ? <Tooltip title={forbidText}>{segmentButton}</Tooltip> : segmentButton}
```
Comment (MMMMoriaty) [inline packages/lb-components/src/views/MainView/toolHeader/SwitchPattern/index.tsx:79]: why don't you just use 'forbidText' without 'forbidSegmentation' ? This allows you saving one param

Comment (Kerwin-L) [reply packages/lb-components/src/views/MainView/toolHeader/SwitchPattern/index.tsx:79]: Good idea！

**Pull request merged**